templates/operations.html

```diff
@@ -615,8 +615,7 @@ <h1 id="time-tactic" class="member-title"></h1>
         }
         document.getElementById('more-modal').style.display='block';
         $('#resultCmd').html(atob($(elem).attr('data-encoded-cmd')));
-        let link_id = opId + '-' + lnk;
-        restRequest('POST', {'index':'result','link_id':link_id}, loadResults);
+        restRequest('POST', {'index':'result','link_id':lnk.toString()}, loadResults);
```

Comment:
Let's make sure the API hand link_id as either an int or string (so normalize on back-end to string)